libraries/chain/include/eosio/chain/apply_context.hpp

```diff
@@ -599,54 +79,25 @@ class apply_context {
       void add_ram_usage( account_name account, int64_t ram_delta );
       void finalize_trace( action_trace& trace, const fc::time_point& start );
 
-      bool is_context_free()const { return context_free; }
-      bool is_privileged()const { return privileged; }
-      action_name get_receiver()const { return receiver; }
-      const action& get_action()const { return *act; }
+      bool is_context_free()const override { return context_free; }
+      const action& get_action()const override { return *act; }
       const action* get_action_ptr()const { return act; }
-      const std::optional<sync_call_context>& get_sync_call_ctx()const { return sync_call_ctx; }
-      std::optional<sync_call_context>& get_mutable_sync_call_ctx() { return sync_call_ctx; }
 
-      action_name get_sender() const;
+      action_name get_sender() const override;
 
       bool is_applying_block() const { return trx_context.explicit_billed_cpu_time; }
       bool is_eos_vm_oc_whitelisted() const;
       bool should_use_eos_vm_oc()const;
```

Comment:
These will need to be moved up to `host_context`. I assume you will do that when you do the eosvmoc work.

Reply:
Yes, they will be moved up when we do the OC work (after EOSVM JIT and Interpreter).